README.md

```diff
@@ -5,17 +5,14 @@ A python wrapper for resin.io [work in progress]
 ## usage 
 
 ```
-pip install resin
```

Comment:
I have the created the project on PyPI, to save the (name) But I will share with apo so you can deploy, when you think its ready for public.